.github/workflows/test.yml

```diff
@@ -39,10 +39,10 @@ jobs:
     - name: Verify generated files
       run: make install-tools generate check-generated
     - name: Run golangci-lint
-      uses: golangci/golangci-lint-action@55c2c1448f86e01eaae002a5a3a9624417608d84  # v6.5.2
+      uses: golangci/golangci-lint-action@dec74fa03096ff515422f71d18d41307cacde373  # v7.0.0
```

Comment:
https://github.com/golangci/golangci-lint-action/releases/tag/v7.0.0 now points to https://github.com/golangci/golangci-lint-action/commit/1481404843c368bc19ca9406f87d6e0fc97bdcfd

Was the tag recreated?

ref: 
- https://github.com/lima-vm/lima/pull/3395#pullrequestreview-2733005804

Reply:
- https://github.com/lima-vm/lima/pull/3413